apps/user-office-backend/src/mutations/ProposalMutations.ts

```diff
@@ -226,6 +226,12 @@ export default class ProposalMutations {
         proposalPk,
       });
     }
+    if (proposal.submitted) {
+      return rejection('Can not submit the proposal again', {

```

Comment:
Text can be improved if possible. "Proposal already submitted "

Reply:
Updated. Please check, thanks.